lib/tasks/gettext.rake

```diff
@@ -83,6 +145,17 @@ namespace :gettext do
     puts "POT file #{pot_file_path} has been generated"
   end
 
+  desc 'Generate POT file for metadata'
+  task :generate_metadata_pot do
+    generate_new_pot_metadata
+    puts "POT file #{metadata_pot_file_path} has been generated"
+  end
+
+  desc 'Merge POT Files'
+  task :merge_pot_files do
+    system("msgmerge -U #{metadata_pot_file_path} #{pot_file_path} --output-file=locales/output.pot")
```

Comment:
the name of the new POT needs to be changed back to `GettextSetup.config['project_name']` at some point. Perhaps just another line or two in this task?